tests/integration-tests/tests/dns/test_dns/test_hit_no_cluster_dns_mpi/print_hostname.sh

```diff
@@ -1,2 +0,0 @@
-#!/bin/bash
-echo `hostname` > /shared/compute_hostname
```

Comment:
Why did you remove it?

Reply:
https://github.com/aws/aws-parallelcluster/pull/2018/files#diff-72b5de0831d42ea5780a01dcadb05167c21810c8c1416993f31a43eaff22eb83R43
This PR forgot to remove it.